### ISSUE TITLE
Webpage is not compliant

### ISSUE THREAD

**Issue by Munesh-Naidu** [Munesh-Naidu/github-issues-templates#5]
Describe the bug
The webpage has a score of zero, which means that it is at risk of accessibility lawsuits.

To Reproduce

Go to 'https://www.accessibilitychecker.org/'
Paste in https://munesh-naidu.github.io/github-issues-templates/mission.html
See errors displayed on the right side of the screen
Expected behavior
The website should have a score of 75%+, to avoid the risk of accessibility lawsuits.

Screenshots
You can view results by clicking this URL:
https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fmunesh-naidu.github.io%2Fgithub-issues-templates%2Fmission.html&flag=us

Desktop (please complete the following information):
-- Chrome

Smartphone (please complete the following information):

N/A
Additional context
Utilize the recommended providers that the site provides, in order to receive a score of 75% or higher.
